tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/checkedexpr/CheckedExpressionOperatorTest.java

```diff
@@ -249,7 +249,7 @@ public void testCheckInBinaryLTExpression() {
     }
 
     @Test(description = "Test service resource that returns an error containing check expression",
-            groups = "brokenOnErrorChange")
+            groups = {"disableOnOldParser", "brokenOnNewParser"})
```

Comment:
Is this test still broken? The expectation is to not have compilation errors right?